tests/browserify-public/browserify-bundle.js

```diff
@@ -1311,17 +1318,6 @@ Interceptor.prototype.replyWithFile = function replyWithFile(statusCode, filePat
     return this.reply(statusCode, readStream, headers);
 };
 
-Interceptor.prototype.replyWithFile = function replyWithFile(statusCode, filePath, headers) {
-    if (! fs) {
-        throw new Error('No fs');
-    }
-    var readStream = fs.createReadStream(filePath);
-    readStream.pause();
-    this.filePath = filePath;
-    return this.reply(statusCode, readStream, headers);
-};
-
-
```

Comment:
As on https://github.com/node-nock/nock/pull/721, I'm not sure why this is here, presumably from a previous change that forgot to commit this.